main.rb

```diff
@@ -0,0 +1,71 @@
+require_relative './music/music_util'
+require_relative './classes/game_app'
+require_relative './books/app'
+
+def main
+  console
+end
+
+def console
+  until options
+    choice = gets.chomp.to_i
+    if choice == 11
+      puts 'Thank You for using my Catalog of Things!!'
+      break
+    end
+    user_input(choice)
+  end
+end
+
+def options
+  puts
+  puts 'List of things'
+  puts '========================'
+  puts '0 Save and Exit'
+  puts '1 List all books'
+  puts '2 List all music albums'
+  puts '3 List all games'
+  puts '4 List all genres'
+  puts '5 List all Labels'
+  puts '6 List all authors'
+  puts '7 List all sources'
```

Comment:
Since you are not doing `sources` in this group, it is unnecessary to have the sources option. Kindly remove it.